tests/ui/test_ui_selenium.py

```diff
@@ -0,0 +1,76 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.framework.testlib import tier1
+from ocs_ci.ocs.ui_selenium.ui_operator import UserInterfaceSelenium
+from ocs_ci.ocs.resources.pvc import get_all_pvc_objs, delete_pvcs
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="module")
+def usr_int(request):
+
+    usr_int = UserInterfaceSelenium()
+    usr_int.ui_login()
+
+    def teardown():
+        usr_int.cleanup()
+        pvc_objs = get_all_pvc_objs(namespace="openshift-storage")
+        pvcs = [pvc_obj for pvc_obj in pvc_objs if "test-pvc" in pvc_obj.name]
+        delete_pvcs(pvc_objs=pvcs)
+
+    request.addfinalizer(teardown)
+    return usr_int
+
+
+class TestPvcUserInterface(object):
+    """
+    Test PVC User Interface
+
+    """
+
+    @tier1
+    @pytest.mark.parametrize(
+        argnames=["sc_type", "pvc_name", "access_mode", "pvc_size"],
+        argvalues=[
+            pytest.param(
+                *["ocs-storagecluster-ceph-rbd", "test-pvc-rbd", "ReadWriteOnce", "1"]
+            ),
+        ],
+    )
+    def test_create_delete_pvc(self, usr_int, sc_type, pvc_name, access_mode, pvc_size):
+        """
+        Test create and delete pvc via UI
+
+        """
+        usr_int.create_pvc_ui(sc_type, pvc_name, access_mode, pvc_size)
+        time.sleep(10)
```

Comment:
Better to avoid sleep here. Check if this can be replaced by some 'wait' function.